linera-views/src/test_utils/mod.rs

```diff
@@ -347,6 +348,21 @@ fn get_random_key_values2(num_entries: usize, len_value: usize) -> Vec<(Vec<u8>,
     key_values
 }
 
+/// Add a prefix to a list of key-values
+pub fn prefix_expansion(
```

Comment:
Nit: I'd suggest renaming this to be a verb, like `expand_prefix` or maybe `prepend_prefix`/`add_prefix`.

Reply:
Ok, for `add_prefix`.